### PR TITLE
Fix: User management table height.

### PR DESCRIPTION
**Pull request by tejaaswini-narendra**

### CODE REVIEW

File: app/client/src/pages/organization/settings.tsx [hunk 1]
```diff
@@ -266,7 +271,7 @@ export const OrgSettings = (props: PageProps) => {
           tableSizes={TABLE_SIZES[CompactModeTypes.DEFAULT]}
           backgroundColor={Colors.ATHENS_GRAY_DARKER}
         >
-          <div className="tableWrap">
+          <div className="tableWrap" style={{ height: `${tableHeight}px` }}>
```
Comment (hetunandu) [inline app/client/src/pages/organization/settings.tsx:274]: You need to pass this into the TableWrapped component as a prop.

File: app/client/src/pages/organization/settings.tsx [hunk 2]
```diff
@@ -182,7 +187,7 @@ export const OrgSettings = (props: PageProps) => {
     const tableDataLength =
       userTableData.length * TABLE_SIZES[CompactModeTypes.DEFAULT].ROW_HEIGHT +
       TABLE_SIZES[CompactModeTypes.DEFAULT].COLUMN_HEADER_HEIGHT;
-    return tableDataLength < 200 ? tableDataLength : 200;
+    return tableDataLength;
```
Comment (hetunandu) [inline app/client/src/pages/organization/settings.tsx:190]: Can we still keep a minimum height for less rows

Comment (tejaaswini-narendra) [reply app/client/src/pages/organization/settings.tsx:190]: There is no minimum height. It will take the height of the table.

Comment (hetunandu) [reply app/client/src/pages/organization/settings.tsx:193]: If the table has no rows.. how does it behave? I would still like a few rows rendered in such a scenario.

Comment (tejaaswini-narendra) [reply app/client/src/pages/organization/settings.tsx:193]: It will have at least one row right because we will have the owner details.

**Pull request merged**